openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.less

```diff
@@ -71,7 +71,7 @@
     padding: 0px;
     margin: 0px;
     line-height: 0px;
-    height: auto;
+    height: 16px;
```

Comment:
@aniketkatkar97, Can you explain why we are adding fixed height?

Reply:
Height auto is working fine for 'Chrome' but it is giving height 0 in the 'Safari' browser. As the button is of type link. I have given it the height same as the default font size.